src/Raven.Server/ServerWide/Commands/ClusterTransactionCommand.cs

```diff
@@ -65,14 +65,22 @@ public static ClusterTransactionDataCommand FromCommandData(BatchRequestParser.C
             }
 
             public DynamicJsonValue ToJson(JsonOperationContext context)
+            {
+                var djv = ToJson();
+                if(Document != null)
+                    djv[nameof(Document)] = Document.Clone(context);
```

Comment:
this is fine? IIRC underneath this was a list of properties, so 'Document' will be twice here?

Reply:
I kept the same behaviour as before my PR.
Would you like me to improve that to avoid duplication?